src/Rule/EachHandler.php

```diff
@@ -58,6 +60,8 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
             }
         }
 
+        $context->setParameter(ValidationContext::PARAMETER_CURRENT_EACH_INDEX, null);
```

Comment:
It's specific `Each` parameter. Suggest move constant to `Each` rule class and rename to `PARAMETER_EACH_KEY`.